data/human/news.txt

```diff
@@ -679,6 +767,8 @@ news "human religious"
 	name
 		word
 			"Mystic"
+			"Priest"
```

Comment:
Given that one of the potential messages here refers to Allah, and Islam doesn't use the term "priest" (an equivalent term might by Imam or Mullah), not sure if we should use religion-specific titles. (And we can't just simply put Imam or Mullah here, as then there's a clash with the Jesus message.) Perhaps "Religious scholar" or "Religious leader" would be better neutral terms.

Alternatively, we could split the references to different religions into different news entries, then scatter them around specific locations as to suggest that certain religions are concentrated in different parts of the galaxy.